test_utils/edx_platform_mocks_shared/fake_models/models.py

```diff
@@ -32,6 +32,7 @@ class CourseOverview(models.Model):
     self_paced = models.BooleanField(default=False)
     course_image_url = models.TextField()
     visible_to_staff_only = models.BooleanField(default=False)
+    effort = models.CharField(max_length=255)
```

Comment:
always remember, please: mocks are our hidden enemy in tests. We need to make similar to the original as much as we can
```suggestion
    effort = models.TextField(null=True)
```